sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/options/AwsModule.java

```diff
@@ -74,6 +73,7 @@
   public static final String MAX_CONNECTIONS = "maxConnections";
   public static final String READ_TIMEOUT = "socketTimeout";
 
+  @SuppressWarnings({"nullness"})
```

Comment:
Please link back to the Jira

Reply:
https://github.com/apache/beam/pull/16752